crates/modelardb_compression/src/models/gorilla.rs

```diff
@@ -110,8 +112,12 @@ impl Gorilla {
         } else {
             // Store each new value as its leading zero bits (if necessary) and
             // meaningful bits (all bits from the first to the last one bit).
-            let leading_zero_bits = value_xor_last_value.leading_zeros() as u8;
+            let mut leading_zero_bits = value_xor_last_value.leading_zeros() as u8;
             let trailing_zero_bits = value_xor_last_value.trailing_zeros() as u8;
+            // Avoid overflow of leading bits.
```

Comment:
Maybe add a newline before the comment, extend the comment to say "leading zero bits", and extend the comment to describe why the "leading zero bits" may overflow if this is not done.